### PR TITLE
[delegation] add statsig to banner

### PR DESCRIPTION
**Pull request by jillxuu**
https://console.statsig.com/6tCucvYJuCUChTKfJsuP2m/dynamic_configs/staking_banner

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -76,7 +76,7 @@ ReactDOM.render(
       }
       waitForInitialization={true}
       options={{
-        environment: {tier: "production"},
+        environment: {tier: process.env.NODE_ENV},
```
Comment (0xZihan) [inline src/index.tsx:79]: how do we want to manage this?

File: src/pages/Validators/Index.tsx [hunk 2]
```diff
@@ -8,14 +11,48 @@ import ValidatorsMap from "./ValidatorsMap";
...
+  // Create a key for storing the view count and last visit timestamp in localStorage
+  const viewCountKey = `${stableID}_view_count`;
+  const lastVisitKey = `${stableID}_last_visit`;
+
+  // Get the current view count and last visit timestamp from localStorage
+  const currentViewCount = localStorage.getItem(viewCountKey);
+  const lastVisitTimestamp = localStorage.getItem(lastVisitKey);
+
+  useEffect(() => {
+    if (viewCountCap && Number(currentViewCount) <= Number(viewCountCap)) {
```
Comment (angieyth) [inline src/pages/Validators/Index.tsx:28]: can `viewCountCap` be null?

**Review by angieyth — APPROVED** [review]
I have no permission to see the statsig console. can you add me in?

File: src/pages/Validators/Index.tsx [hunk 3]
```diff
@@ -8,14 +11,48 @@ import ValidatorsMap from "./ValidatorsMap";
...
+  }, []);
 
   return (
     <Box>
       <PageHeader />
       <Typography variant="h3" marginBottom={2}>
         Validators
       </Typography>
-      <StakingBanner />
+      {currentViewCount && Number(currentViewCount) <= 5 ? (
```
[the remark this answers is not in the document]
Comment (jillxuu) [reply src/pages/Validators/Index.tsx:53]: ahh good catch! should use `viewCountCap`!

File: src/pages/Validators/Index.tsx (hunk 2 above)
Comment (jillxuu) [reply src/pages/Validators/Index.tsx:28]: yes, if `viewCount` is not not set in statsig or we couldn't find the config itself

**Pull request merged**